noir-projects/aztec-nr/easy-private-state/src/easy_private_uint.nr

```diff
@@ -24,19 +24,28 @@ impl<Context> EasyPrivateUint<Context> {
 impl<Context> EasyPrivateUint<&mut PrivateContext> {
     // Very similar to `value_note::utils::increment`.
     pub fn add(self, addend: u64, owner: AztecAddress, outgoing_viewer: AztecAddress) {
-        let owner_npk_m_hash = get_current_public_keys(self.context, owner).npk_m.hash();
+        let owner_keys = get_current_public_keys(self.context, owner);
+        let outgoing_viewer_keys = get_current_public_keys(self.context, outgoing_viewer);
```

Comment:
I imagine this will soon be a very common pattern unfortunately. Maybe we can store this somewhere down the line to reduce boilerplate. But not worth looking into atm given key rotation is up in the air.